src/main/java/List.java

```diff
@@ -0,0 +1,102 @@
+public class List {
+
+    private final int MAX_INSTRUCTION = 100;
+    private Task[] instruction = new Task[MAX_INSTRUCTION];
+    private int index = 0; // Array index
```

Comment:
rather than using "index" as the variable to track the last element of the array, perhaps using "numberOfInstructions" may be more intuitive to understand.